temporalcli/commandsmd/commands.md

```diff
@@ -1311,6 +1311,9 @@ temporal server start-dev \
 * `--ui-ip` (string) -
   IP address bound to the WebUI.
   Default is same as '--ip' value.
+* `--public-path` (string) -
```

Comment:
```suggestion
* `--ui-public-path` (string) -
```

Actually, I'm sorry to just now notice this, but can you add the `ui-` prefix to the arg?